pkg/exporters/k8sexporter/condition/manager.go

```diff
@@ -36,10 +37,17 @@ const (
 	updatePeriod = 1 * time.Second
 	// resyncPeriod is the period at which condition manager does resync, only updates when needed.
 	resyncPeriod = 10 * time.Second
+)
+
+var (
 	// heartbeatPeriod is the period at which condition manager does forcibly sync with apiserver.
-	heartbeatPeriod = 1 * time.Minute
+	heartbeatPeriod time.Duration
 )
 
+func init() {
+	flag.DurationVar(&heartbeatPeriod, "heartbeat-period", 1 * time.Minute, "heartbeatPeriod is the period at which condition manager does forcibly sync with apiserver")
```

Comment:
Can you add an option to NodeProblemDetectorOptions and create the flag in:
https://github.com/kubernetes/node-problem-detector/blob/master/cmd/options/options.go

Reply:
Done.